### PR TITLE
Feature/20 manpage with GitHub

### PR DESCRIPTION
**Pull request by SchleichsSalaticus**

### CODE REVIEW

File: .github/workflows/manpage-build.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Pages
+on:
+  push:
+    branches:
+    - main
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
```
Comment (tomschr) [inline .github/workflows/manpage-build.yml:9]: Just a tip: I find it much more readable if you insert a linebreak between each step. This makes each step much more readable as it is shown as one unit.

**Review by tomschr — COMMENTED** [review]
@SchleichsSalaticus Good job! Just some minor comments.

Maybe you should additionally include a `docs/requirements.txt` file ~with the Sphinx extension in it.~ This file will be used when installing the Python environment.

Enable it in the workflow file with the `install_requirements` parameter in your second step.

**Update**: It seems, the `sphinx.ext.githubpages` extension is a builtin extension. So no third party library needed. Nevertheless, I consider the addition with the `docs/requirements.txt` file as useful.

File: .github/workflows/manpage-build.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Pages
```
[the remark this answers is not in the document]
Comment (SchleichsSalaticus) [reply .github/workflows/manpage-build.yml:1]: Thanks for the advice, this will help a lot

**Review by tomschr — APPROVED** [review]
Looks great! :+1: Thank you!

**Pull request merged**